pages/index.vue

```diff
@@ -1989,7 +1989,7 @@ export default {
   },
   mounted() {
     this.observeRequestButton();
-    this._keyListener = function(e) {
+    this._keyListener = (e)=> {
```

Comment:
@jamesgeorge007 What abt these arrows? Will they work correctly?

Reply:
I guess they're gonna introduce scope resolution ambiguities

Reply:
English please 😅

Reply:
We can't use arrow operator on any functions involving `this` since it's scope is targeted to that function, not global vars.

Reply:
😿